src/CodeStyle/Core/CodeFixes/Microsoft.CodeAnalysis.CodeStyle.Fixes.csproj

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <Compile Include="..\..\..\Features\Core\Portable\CodeFixes\PredefinedCodeFixProviderNames.cs" Link="PredefinedCodeFixProviderNames.cs" />
     <Compile Include="..\..\..\Workspaces\Core\Portable\Shared\Utilities\IProgressTracker.cs" Link="IProgressTracker.cs" />
+    <Compile Include="..\..\..\Workspaces\Core\Portable\CodeFixes\FixAllOccurrences\DocumentBasedFixAllProvider.cs" Link="DocumentBasedFixAllProvider.cs" />
```

Comment:
why?

Reply:
➡️ The original implementation was based on this approach. We already had the type in the code base so it was linked it to support the new code fix.

Reply:
I'm not actually seeing FixAll tests, so i'm not sure how to evaluate this.  It's also unclear to me why this is in src/CodeStyle/Core since  the feature doesn't appear to be there.

Reply:
`DocumentBasedFixAllProvider` was already in CodeStyle.Fixes. I moved it to Workspaces to make it accessible to the fixAll and other future fixAlls. I could have alternately refernced CodeStyle.Fixes from the CSharp.EditorFeatures project.

Reply:
> CodeStyle.Fixes

This is not currently possible due to #30472